src/frontend/komponenter/Fagsak/Simulering/Simulering.tsx

```diff
@@ -100,21 +103,37 @@ const Simulering: React.FunctionComponent<ISimuleringProps> = ({ åpenBehandling
                         <SimuleringPanel simulering={simuleringsresultat.data} />
                         <SimuleringTabell simulering={simuleringsresultat.data} />
                         {(!erMigreringMedStoppISimulering ||
+                            erMigreringMedFeilutbetalingInnenforBeløpsgrenser ||
                             skalIkkeStoppeMigreringsbehandlinger) &&
                             erFeilutbetaling && (
-                                <TilbakekrevingSkjema
-                                    søkerMålform={hentSøkersMålform(åpenBehandling)}
-                                    harÅpenTilbakekrevingRessurs={harÅpenTilbakekrevingRessurs}
-                                />
+                                <>
+                                    {erMigreringMedFeilutbetalingInnenforBeløpsgrenser && (
+                                        <>
+                                            <br />
+                                            <Alert variant="warning" size="medium">
+                                                Behandlingen medfører en feilutbetaling. Ved
+                                                feilutbetalingsbeløp på mindre enn totalt 40 kroner,
+                                                kan du gå videre i behandlingen. Du må huske å sende
+                                                oppgave til NØS om at det ikke skal opprettes
+                                                kravgrunnlag.
+                                            </Alert>
+                                        </>
```

Comment:
Her hadde det vært diggere å lage en StyledBeløpsgrenseAlert med `margin-top: 1rem` i stedet for å bruke `br` til å lage spacingen 🙂 
```js
const StyledBeløpsgrenseAlert = styled(Alert)`
    margin-top: 1rem;
`
```
```js
{erMigreringMedFeilutbetalingInnenforBeløpsgrenser && (
    <StyledBeløpsgrenseAlert variant="warning" size="medium">Innhold</StyledBeløpsgrenseAlert>
)}
```